database/process_db_test.go

```diff
@@ -0,0 +1,49 @@
+package database
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/mesg-foundation/engine/hash"
+	"github.com/mesg-foundation/engine/process"
+	"github.com/stretchr/testify/require"
+)
+
+func TestProcessDB(t *testing.T) {
+	dir, _ := ioutil.TempDir("", "process.db.test")
+	defer os.Remove(dir)
+
+	db, err := NewProcessDB(dir)
+	require.NoError(t, err)
+	defer db.Close()
+
+	p := &process.Process{
```

Comment:
Could you add more process with different data?
like one with only hash and another one with ALL:
```
var testProcess = &process.Process{
	Hash: hash.Int(1),
	Key:  "key",
	Edges: []*process.Process_Edge{
		&process.Process_Edge{
			Src: "dw",
			Dst: "dw",
		},
	},
	Nodes: []*process.Process_Node{
		&process.Process_Node{
			Type: &process.Process_Node_Result_{
				Result: &process.Process_Node_Result{
					Key:          "foo",
					InstanceHash: hash.Int(1),
					TaskKey:      "taskKey",
				},
			},
		},
	},
}
```